### PR TITLE
neturl can't parse correctly without scheme

### PR DESCRIPTION
**Pull request by jialeicui**

### CODE REVIEW

File: lualib/ourl/init.lua [hunk 1]
```diff
@@ -95,6 +95,11 @@ end
 
 local function url_modify(url, scheme)
     scheme = scheme or 'http'
+    
+    if not ngx.re.match(url, '^https?://*', 'o') then
```
Comment (xiaooloong) [inline lualib/ourl/init.lua]: the regex here, do you mean `^https?:/+` or `^https?://.*` ?

**Review by xiaooloong — COMMENTED** [review]
`^https?:/+` or `^https?://.*` ?

File: lualib/ourl/init.lua (hunk 1 above)
Comment (jialeicui) [reply lualib/ourl/init.lua]: sorry, my fault, changes as below

**Comment by xiaooloong** [comment]
Thanks for your contribution :D

**Pull request merged**